shared/kubernetes/provider.go

```diff
@@ -651,7 +651,17 @@ func (provider *Provider) ApplyMizuTapperDaemonSet(ctx context.Context, namespac
 	agentContainer.WithName(tapperPodName)
 	agentContainer.WithImage(podImage)
 	agentContainer.WithImagePullPolicy(imagePullPolicy)
-	agentContainer.WithSecurityContext(applyconfcore.SecurityContext().WithPrivileged(true))
+
+	caps := applyconfcore.Capabilities().WithDrop("ALL").WithAdd("NET_RAW")
```

Comment:
I think we need to add `NET_ADMIN` as well because of the `-promisc` flag.

> `NET_RAW` permits raw access to an interface for capturing directly off the wire (by default enabled in Kubernetes)
> `NET_ADMIN` allows us to set an interface to promiscuous mode (required by the `-promisc` flag in the tapper)
> See `CAP_NET_` prefixed Linux capabilities for more https://man7.org/linux/man-pages/man7/capabilities.7.html

Related PR https://github.com/up9inc/mizu/pull/436

Reply:
It's interesting, I wonder how come it worked without it, maybe it has something to do with kernel versions (tested with `5.15.4`).

I'll add the NET_ADMIN because:
- Istio need it as well
- To be on the safe side

Reply:
@dvdlevanon great!